packages/common/container-definitions/src/loader.ts

```diff
@@ -389,21 +389,48 @@ export interface IContainer extends IEventProvider<IContainerEvents> {
 	proposeCodeDetails(codeDetails: IFluidCodeDetails): Promise<boolean>;
 
 	/**
-	 * Attaches the Container to the Container specified by the given Request.
+	 * When a container is created in Fluid it starts in what we call a detached state,
+	 * which means it is detached from server storage and unavailable for other users to open or collaborate with.
```

Comment:
Avoid using the term in the definition
```suggestion
	 * which means it is not yet uploaded to server storage and unavailable for other users to open or collaborate with.
```